CONTRIBUTING.md

```diff
@@ -3,62 +3,75 @@
 #### Useful Resources
 
 - [AWS Step Functions Developer Guide](https://docs.aws.amazon.com/step-functions/latest/dg/welcome.html)
+- [Amazon States Language](https://states-language.net/)
+- [AWS Serverless Application Model](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/what-is-sam.html)
+- [AWS SAM CLI Command Reference](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-command-reference.html)
+- [FFmpeg Static Builds](https://www.johnvansickle.com/ffmpeg/)
+- [FFmpeg CLI Command Reference](https://ffmpeg.org/ffmpeg.html)
 
-## Development Environment
+## Project Standards & Guidelines
 
-If the IDE you use supports per-repository settings (such as with the `.vscode` directory for VS Code), you should add and check in those settings. Please ensure that the settings match those described in this document.
+Porter integrates a number of different services, technologies, frameworks and programming languages. In order to help with project maintenance and reducing mental overhead when working on the project, please follow the standards and guidelines described in this section. Whenever possible, ensure that any new or changed standards are documented here and are enforced automatically, such as by tests and IDEs.
 
-Any code styles that this repository follows should be described in this document, and should be checked by CI.
+### Security
+
+Take a _least privilege_ approach to all aspects of security. Create discrete IAM roles for each Lambda function, Fargate task, etc, and only give the role the permissions it needs to operate. Use the private VPC subnets with VPC endpoints unless it's absolutely necessary that something have access to the public internet.
+
+### Git & Source Control
+
+Git commits always should follow the [seven rules](https://chris.beams.io/posts/git-commit/#seven-rules).
```

Comment:
+1 ... I also sometimes try to squash-merge a PR with a bunch of churn, and give it a nicer/uniform commit message.